geode-core/src/distributedTest/java/org/apache/geode/cache/ConnectionPoolDUnitTest.java

```diff
@@ -120,24 +119,14 @@ public void setup() {
 
   @After
   public void tearDown() {
-    getSystem();
-    Invoke.invokeInEveryVM("getSystem", new SerializableRunnable() {
-      @Override
-      public void run() {
-        getSystem();
-      }
-    });
-  }
-
-  @Override
-  public final void postTearDownCacheTestCase() {
     Invoke.invokeInEveryVM(() -> {
-      Map<String, Pool> pools = PoolManager.getAll();
-      assertThat(pools).describedAs("found pools remaining after teardown: " + pools).isEmpty();
+      if (basicGetCache() != null) {
```

Comment:
Part of this fix was to cleanout the teardown code so it would run correctly.

Reply:
I learned today that you can just do:
```
PoolManager.close()
```
And that'll close all pools.

Reply:
Sweet, next time I will use that...